src/adapters/mc/asb/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# SO binary name must match the class name in MOF with a "lib" prefix added
+project(OsConfigResource)
```

Comment:
Cna we also reflect the project name to be equivalent to the target name `project(OsConfigResourceAsb)`

Reply:
What would this give us? For now this builds fine as-is on Ubuntu 14 where is needed. Let's do improvements later when time. Like making a macro for the ZIP lines, etc.